Cargo.toml

```diff
@@ -34,6 +36,7 @@ domain = { git = "https://github.com/NLnetLabs/domain.git", branch = "main", fea
     "tsig",
     "unstable-client-transport",
     "unstable-sign",
+    "unstable-stelline",
```

Comment:
@ximon18 this is the "controversial" change. If you agree with this then its fine with me to merge it.

Reply:
I see.

The fuzz binary uses FakeEnv which causes this problem.

Could another way be to `impl Env for FuzzEnv` where `FuzzEnv` doesn't use Stelline (and so wouldn't be suitable for running `dnst update` or `dnst notify` unless some non-Stelline based impl of `Env::dgram()` and `Env::stub_resolver_from_confg()` would be possible)?

Reply:
I wonder if we should do ldns-testns. If so then we will get stelline as well.

Reply:
Are you referring to https://github.com/NLnetLabs/dnst/pull/56?

Reply:
Yes. We may need to rethink the name and design for the dnst version. But it seems a useful feature.

Reply:
Interesting idea... :-)